### PR TITLE
Add front view notification for admin if carousel bails

### PR DESCRIPTION
**Pull request by Luegge**
As proposed in https://github.com/Automattic/jetpack/issues/1023, this adds a front end notification for the admin if the carousel module bails.
The style is identical to the `.update-nag` class in the admin back end.

Potential problems: Priorities from custom [gallery] shortcode override functions. This implementation uses the default '10' priority.

### CODE REVIEW

**Comment by Luegge** [comment]
Any updates?

**Comment by samhotchkiss** [comment]
Hey @Luegge -- looks like this needs to be rebased-- I'm putting it on the docket to get checked out for 3.5, please rebase at some point in the next couple of weeks, thanks!

**Comment by Luegge** [comment]
Thanks for the reply, @samhotchkiss. Rebase should be done. Tell me if anything went wrong, it was my first rebase :-)

**Comment by samhotchkiss** [comment]
Looks like it was a successful one!  We'll check this out during the next release cycle.  Thanks, @Luegge !

File: modules/carousel/jetpack-carousel.php [hunk 1]
```diff
@@ -77,12 +77,26 @@ function jetpack_configuration_load() {
 	function asset_version( $version ) {
 		return apply_filters( 'jp_carousel_asset_version', $version );
 	}
-
+	
+	function display_bail_message( $output= '' ) {
+		// Displays a message on top of gallery if carousel has bailed
+		$message = '<div class="jp-carousel-msg"><p>';
+		$message .= __( 'The carousel plugin has been disabled, because someone is overriding the [gallery] shortcode.', 'jetpack' );
```
Comment (dereksmart) [inline modules/carousel/jetpack-carousel.php]: I think I prefer "another plugin or theme" rather than "someone"

**Comment by dereksmart** [comment]
Looks reasonable :+1: unsure about verbiage though

**Comment by Luegge** [comment]
Thanks for the review. I changed the message accordingly, since it does make more sense. I guess my initial message was based on the comment in the file that sais "Bail because someone is overriding...".

**Pull request merged**